packages/cx/src/charts/Swimlanes.js

```diff
@@ -33,6 +33,8 @@ export class Swimlanes extends BoundedObject {
       let { bounds } = data;
       let { CSS, baseClass } = this;
 
+      if (data.step < 0 || data.size < 0) return;
```

Comment:
It would be better to use <= for both checks, because of the potential division by 0.

Reply:
Fixed